src/lib.rs

```diff
@@ -516,17 +516,17 @@ impl<'b, T: Write + 'b> Session<'b, T> {
 pub(crate) fn create_emitter<'a>(config: &Config) -> Box<dyn Emitter + 'a> {
     match config.emit_mode() {
         EmitMode::Files if config.make_backup() => {
-            Box::new(emitter::FilesWithBackupEmitter::default())
+            <Box<emitter::FilesWithBackupEmitter>>::default()
         }
         EmitMode::Files => Box::new(emitter::FilesEmitter::new(
             config.print_misformatted_file_names(),
         )),
         EmitMode::Stdout | EmitMode::Coverage => {
             Box::new(emitter::StdoutEmitter::new(config.verbose()))
         }
-        EmitMode::Json => Box::new(emitter::JsonEmitter::default()),
-        EmitMode::ModifiedLines => Box::new(emitter::ModifiedLinesEmitter::default()),
-        EmitMode::Checkstyle => Box::new(emitter::CheckstyleEmitter::default()),
+        EmitMode::Json => <Box<emitter::JsonEmitter>>::default(),
+        EmitMode::ModifiedLines => <Box<emitter::ModifiedLinesEmitter>>::default(),
+        EmitMode::Checkstyle => <Box<emitter::CheckstyleEmitter>>::default(),
```

Comment:
I feel it's not better than before. 🤔

Reply:
Certainly not easier to understand, but maybe a bit more efficient in a future version of Rust. But you're right, it doesn't seem to be worth it.